# The gesturefile to be executed.
bool move_exec_gesturefile

---
# Whether the CommU added the gesture is executed successfully.
bool success
